packages/compass-crud/src/stores/crud-store.js

```diff
@@ -698,7 +677,7 @@ const configureStore = (options = {}) => {
       }
 
       const csfleState = { state: 'none' };
-      const dataServiceCSFLEMode = this.dataService.getCSFLEMode && this.dataService.getCSFLEMode();
+      const dataServiceCSFLEMode = this.dataService && this.dataService.getCSFLEMode && this.dataService.getCSFLEMode();
```

Comment:
For some reason a test started failing because that one test did some extensive mocking and the store doesn't have a dataService. Now that the onInstanceCreated code actually executes it got here and blew up because dataService doesn't exist in that case.

Not crazy about this kind of thing where we aren't strict about what should exist and then use proper test helpers to make sure everything exists, but for now I think I can live with this.